program/idl.json

```diff
@@ -0,0 +1,164 @@
+{
+  "version": "0.0.0",
+  "name": "yellowstone_blocklist",
+  "instructions": [
+    {
+      "name": "Create",
+      "accounts": [
+        {
+          "name": "mint",
+          "isMut": false,
+          "isSigner": false,
+          "docs": [
+            "The token extensions mint account linked to the policy"
+          ]
+        },
+        {
+          "name": "tokenAccount",
+          "isMut": false,
+          "isSigner": false,
+          "docs": [
+            "The authority over the policy based on token onwership of the mint"
+          ]
+        },
+        {
+          "name": "policy",
+          "isMut": true,
+          "isSigner": false,
+          "docs": [
+            "The blocklist policy account"
+          ]
+        },
+        {
+          "name": "payer",
+          "isMut": true,
+          "isSigner": true,
+          "docs": [
+            "The account paying for the storage fees"
+          ]
+        },
+        {
+          "name": "systemProgram",
+          "isMut": false,
+          "isSigner": false,
+          "docs": [
+            "The system program"
+          ]
+        },
+        {
+          "name": "tokenProgram",
+          "isMut": false,
+          "isSigner": false,
+          "docs": [
+            "The token program"
+          ]
+        }
+      ],
+      "args": [],
+      "discriminant": {
+        "type": "u8",
+        "value": 0
+      }
+    }
+  ],
+  "accounts": [
+    {
+      "name": "Policy",
+      "type": {
+        "kind": "struct",
+        "fields": [
+          {
+            "name": "allowed",
+            "type": {
+              "vec": "publicKey"
```

Comment:
in the previous versio nof the porgram we had a polciy Allow and a policy Deny instead of two separate lists. In my head it is clearer rather than a mixed list like this.

It also make it comparatively easier to predict if you combine two differnet lists in one call.

Reply:
I set back to having a strategy of allow or deny.